src/__tests__/__snapshots__/parseWithPointers.ts.snap

```diff
@@ -1,5 +1,2805 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`json parser fixtures parses main.oas2.json 1`] = `
```

Comment:
Yeah I know... 
These snapshots served as a good way to detect potential regressions. They contain the result of the **former** parser.

Reply:
No problem they make sense.